ingestion/src/metadata/ingestion/source/database/common_db_source.py

```diff
@@ -129,11 +129,16 @@ def get_database_names(self) -> Iterable[str]:
         Sources with multiple databases should overwrite this and
         apply the necessary filters.
         """
-        custom_database_name = self.service_connection.__dict__.get("databaseName")
-
-        database_name = self.service_connection.__dict__.get(
-            "database", custom_database_name or "default"
+        custom_database_name = self.service_connection.__dict__.get(
+            "databaseName", None
```

Comment:
why do we need to return `None` in case of not found by default get returns `None` if key was not found

Reply:
Yes 👍